addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -720,8 +720,9 @@
 		
 		"1104"	//Air Strike
 		{
-			"desp"			"Air Strike: {positive}Every 200 damages gains 1 clip size"
+			"desp"			"Air Strike: {positive}Every 200 damages gains 1 clip size, +30% explosion radius"
 			"tags"			"damage_head_required ; 200"
+			"attrib"		"99 ; 1.2"
```

Comment:
This is now more confusing, +20% bonus (99) was added while -10% penalty (100) still stays, making overall bonus either +8% or +10% depending on math.

I'm not sure whenever if most people would read this desp as -10% penalty is also part of the percentage or ignored.

Reply:
Indeed, it's confusing. 
Changed description to mention +20% bonus while +33% is being applied for rounding, +20% radius splash **compared** to default rocket launcher, and 30% compared to old Air strike